spyder_remote_services/app.py

```diff
@@ -0,0 +1,118 @@
+"""Jupyter server example application."""
```

Comment:
```suggestion
"""Spyder server application."""
```

The current docstring doesn't seem right.